src/util/getFirstFocusable.test.ts

```diff
@@ -0,0 +1,12 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+jest.mock('@uifabric/utilities/lib/focus', () => ({
+  getFirstFocusable: jest.fn(),
+}));
```

Comment:
Why mock it? If you replace it with a spy, and then check that it's a function, it's always going to return true! I could go into the implementation file, replace its contents with `export const getFirstFocusable = 42` and the tests would still pass.

Reply:
Good point.  I get a 'Jest encountered an unexpected token' error without mocking, the linter complains about an incorrect header in `@uifabric/utilities/lib/focus.js`

Will adding `transformIgnorePatterns` to YamUI's jest config fix this?